src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade enables staking withdrawals on Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards are automatically distributed every few days
+  - Validators who fully exit staking will receive their remaining balance
+---
+
+<UpgradeStatus dateKey="page-upgrades-withdrawals">
+  Staking withdrawals will be enabled through the Shanghai/Capella upgrade. This Ethereum network upgrade is expected to take place in the first half of 2023. <a href="#when">More below</a>
+</UpgradeStatus>
+
+The Shanghai/Capella upgrade enables **staking withdrawals** on Ethereum, allowing people to unlock ETH staking rewards. Reward payments will automatically and regularly be sent to a provided withdrawal address linked to each validator. Users can also exit staking entirely, unlocking their full validator balance.
+
+## Staking rewards {#staking-rewards}
+
+Reward payments are automatically processed for active validator accounts with a maxed out effective balance of 32 ETH.
+
+Any balance above 32 ETH earned through rewards does not actually contribute to principle, or increase the weight of this validator on the network, and is thus automatically withdrawn as a reward payment every few days. Aside from providing a withdrawal address one time, these rewards do not require any action from the validator operator. This is all initiated on the consensus layer, thus no gas (transaction fee) is required at any step.
+
+### How did we get here? {#how-did-we-get-here}
+
+Over the past few years Ethereum has undergone several network upgrades transitioning to a network secured by ETH itself, instead of energy-intensive mining as it once was. Participating in consensus on Ethereum is now known as "staking", as participants have voluntarily locked up ETH, placing it "at stake" for the ability to participate in the network. Users who follow the rules will be rewarded, while attempts to cheat can be penalized.
+
+Since the launch of the staking deposit contract in November 2020, some brave Ethereum pioneers have voluntarily locked funds up to activate "validators", the accounts that have the right to formally attest to and propose blocks, following network rules.
+
+Before the Shanghai update, you couldn't use or get your money out if you staked ETH. But now, you can opt-in to automatically receive your rewards into a provided account, and you can also take out all your money whenever you want.
+
+### How do I prepare? {#how-do-i-prepare}
+
+<WithdrawalsTabComparison />
```

Comment:
Same concern as before related to the ease of translation management.
It's not a blocker, but I think it's worth taking the time to consider later how to pass the content to this component from the .md file, instead of having the text living inside the component.

Reply:
Agree. Plan is to publish this and then tee it up for translation. When we do that, we should look at this component and consider ways to improve it for this purpose.